cypress/integration/single-pdf/render-app.ts

```diff
@@ -0,0 +1,15 @@
+describe('Renders single PDF', () => {
+  it('Render page content', () => {
+    cy.loadPdf('/pdf');
+
+    cy.log('check that all the essential buttons are on the page');
+    cy.findByRole('link', { name: 'Return to Homepage' }).should('exist');
+    cy.findByRole('button', { name: 'Table of Contents' }).should('exist');
+    cy.findByRole('button', { name: 'Settings' }).should('exist');
+    cy.findByRole('button', { name: 'Toggle Fullscreen' }).should('exist');
+    cy.findByRole('button', { name: 'Next Page' }).should('exist');
+    cy.findByRole('button', { name: 'Previous Page' }).should('exist');
+
+    cy.findByText('Julius H. Schoeps').should('be.visible');
```

Comment:
I'm surprised that this doesn't time out because the PDF takes so long to load -- would it make sense to switch this out for a smaller pdf if we find one?